### PR TITLE
🌐 sv_SE | Small fix

### PR DESCRIPTION
**Pull request by dennziil**

### CODE REVIEW

File: src/modules/missionHelper/i18n/sv_SE.json [hunk 1]
```diff
@@ -5,94 +5,88 @@
...
-		"max_possible_prisoners": "Maximala fångar",
-		"title": "Fånge | {n} Fånge | {n} Fångar"
+		"max_possible_prisoners": "Maximalt antal fångar",
+		"title": "Fånge | {n} fånge | {n} fångar"
 	},
 	"requirements": {
-		"water_needed": "Vattenbehov (gal)"
+		"water_needed": "Water demand (gal)"
 	},
-	"tip": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/sv_SE.json:38]: Why did you remove the "tip" translations?

File: src/modules/missionHelper/i18n/sv_SE.root.json [hunk 2]
```diff
@@ -30,151 +25,147 @@
...
+				"caption": "Name of the requirement",
+				"description": "Define what the vehicle requirements should be sorted by.",
+				"percentage": "Probability",
+				"title": "Sorting the vehicle requirements"
 			},
 			"title": {
-				"description": "Visar rubriken \"Fordon\" ovanför fordonets krav.",
-				"title": "Fordon (titel)"
-			},
-			"xAfterNumber": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/sv_SE.root.json:175]: Why did you remove ``xAfterNumber``

**Review by JRH-1997 — CHANGES_REQUESTED** [review]
What's the reason that you changed translation back to English? 
Translations that are not needed ingame could be removed from the file.

**Comment by dennziil** [comment]
Did the translate all over again. Used the nb_NO version since it looked a like as the swedish one, more than the en_US does. :)

**Comment by JRH-1997** [comment]
> 
> 
> Did the translate all over again. Used the nb_NO version since it looked a like as the swedish one, more than the en_US does. :)

Yeah, the only thing is that nb_NO is not up-to-date.  Only up-to-date versions are de_DE, all en_** and nl_NL.

**Review by jxn-30 — APPROVED** [review]
